### ISSUE TITLE
True minimum viable governance via EF

### ISSUE THREAD

**Issue by lirazsiri** [gitcoinco/ERC-1789#48]
Discussing on-chain governance at this point is an interesting intellectual exercise but it is so far away from being a realistic solution to any of Ethereum’s problems that our scarce attention would be better spent coming up with answers to better questions. Assuming what we're trying to solve is truly the minimum viable governance for an inflation treasury we can implement ASAP (e.g., with ProgPOW) to increase the amount of resources that goes into vital development that can’t be funded any other way.

Coming up with better questions is often more impactful than coming up with good answers to bad questions.

Maybe the question shouldn’t be: what is the minimum viable **on-chain** governance solution to administer funds from an inflation treasury?

Maybe the question should be: what is the minimum viable governance solution. Full stop.

It doesn't have to be on-chain. On-chain governance solutions are still in their infancy. The most advanced of them are still more utopian science experiment than anything we can sanely risk pouring a significant amount of the community's public resources into at this stage.

A much more practical off-chain solution would be to leverage EF, the most trusted organization in crypto. The EF doesn’t have to be perfect to be far superior to any untested on-chain solution. 

We don’t need to discuss dismantling the EF, as it is already in the process of dismantling itself by giving out unsustainable grants with no expectation of return. Developing public goods through donations suffers from the free rider problem. It penalizes the generous while rewarding the free riders. Essentially the generous are donating their wealth indirectly to the free riders while only capturing an ever decreasing portion of that value. For example, let’s say a generous non-profit holds 2% of ETH tokens and donates 1% of them to developing Ethereum v2. If that generates $100M worth of value for all ETH token holders the non-profit now has $1M worth of value and all the free riders get the other $99M. Within the grant model the break even on investing in Ethereum v2 is 100X ROI.

Anyone that has ETH has the same financial interests as those in control of the EF treasury and yet the vast bulk of funding that goes towards developing Ethereum infrastructure comes from only one organization. Within the current model the EF is like a parent donating all of its blood to its children while attracting predators and parasites attracted by the smell of blood.

Since the EF is already on the path to self-extinction and we don’t have a credible alternative, what we should be discussing right now is:

1) How we can all help support an inflation treasury that saves the EF from its own unsustainable generosity? So the EF can help Ethereum develop to the point where a proven on-chain solution can credibly replace most of what it does.

2) How we can help the EF improve so that it is better at maximizing positive impacts? 

3) How we can help maintain & strengthen the trust between the EF and the Ethereum community?

The community should encourage discussion about what principles would maximize the trust needed for this to work. 

For example, it is reasonable to expect the EF to provide the transparency required to allow the community to vet how inflation treasury funds are spent. We shouldn’t just make it possible to vet inflation treasury expenses we should be actively encouraging vetting by using some of those funds for auditing. We could also post lucrative whistleblower bounties that would go to anyone who manages to uncover evidence for corruption or embezzlement of funds in any form (e.g., fraudulent invoices).

Since the EF is the only realistically viable option within the short-mid term for administering the funds from an inflation treasury, without this trust we won’t get consensus on changing the rules of consensus to even add an inflation treasury to Ethereum in the first place. This would be a tragedy. Ethereum is still very much a work in progress and without an inflation treasury Ethereum doesn’t have a sustainable model for funding “clean air” type infrastructure projects that can’t be monetized such as layer 1 development. 

People who are undermining trust in the only credible opportunity we have to make Ethereum development sustainable in the near term should be criticized for the damage they are causing to our common interest.

Anyone who tells you otherwise is discrediting themselves in a desperate bid for power. 

The timeless problem of politics is that the people who want power the most and are willing to campaign most aggressively to get it are the people that can be trusted the least. Being a power hungry self promoter has historically correlated very poorly with good leadership. If we can trust anyone with power it’s those that want it the least. Those who would rather live quiet lives away from the spotlight. Those who would much rather manufacture real impacts than manufacture fake perceptions. The political game is not for them. Sadly, that’s usually the people who are least likely to end up in power. Somehow by some bizarre twist of fate we’ve been blessed by more than one of them.

Regarding on-chain governance, we should encourage experimentation and support risk taking by those who are bold enough to not just talk but actually work on credible new ways of coordinating groups towards common goals. We should be paying more attention to those who prove their merit by work, not by talk. Talk is cheap. Show me the code. 

Coming up with new models should be done with humility that gives proper respect to what has worked best in the past and does not propose throwing out the baby with the bathwater in a utopian version of the Not Invented Here syndrome.

What we want to encourage in our community is a culture of science. What was revolutionary about science was not a breakthrough of knowledge but a breakthrough in humility. All true human progress is rooted in the ability to admit that we do not know and that conviction is not equivalent to truth. It is all too easy to fool ourselves. That doesn’t mean that truth is forever out of reach, just that we need better ways to test ideas. We should be skeptics, not cynics.

Bottom line, we need to make the most of our off-chain institutions because if we try doing new on-chain governance models on a big scale before proving they work on a small scale the default will be failure. There are many more ways to do something wrong than to do it right. If we try judging what will work based on how intuitively appealing it sounds to most people participating in the consensus process that is also guaranteed to fail. 

Let's say we define "governance" expertise as expertise in coming up with models that have a proven track record of generating sustainable positive impacts. These would be the true political scientists of governance. At this stage there is no evidence that we have genuine scientists among us and certainly no evidence that the community could tell them apart from all the false pseudo-scientists out there who's main skill is pandering to their audience and entertaining the community with political theater. The worst of these are blockchain politicians are wolves in sheep's clothing who are not truly campaigning for the community's best interests but campaigning for their share of the lucrative inflation treasury pie and the power and prestige they hope will come with it.

Without strong measures of performance for what constitutes good governance the community will never be able to tell the difference between the true scientists of governance and the psuedo-scientists.

Here’s to hoping an inflation treasury helps Ethereum get more good engineering and good science. Less political theater. We are at a crossroads.

**Comment by glauseWilde** [comment]
Multi-sig managed faucet. With members of the EF on the multi-sig bootstraps trust from the EF. 

If the faucet decides to point a portion of funds to the EF I have no problem as long as the distribution is handled transparently.

>Bottom line, we need to make the most of our off-chain institutions because if we try doing new on-chain governance models on a big scale before proving they work on a small scale the default will be failure. There are many more ways to do something wrong than to do it right. If we try judging what will work based on how intuitively appealing it sounds to most people participating in the consensus process that is also guaranteed to fail.

The locations of the faucet can be off chain orgs that have all of the prospects you desire. No need for the faucet to be a big scale anything. A small group of 6-12 people with as simple of a governance mechanism as you can get, a minimum number of members signing changes to what accounts receive what proportion of the faucet.

**Comment by lirazsiri** [comment]
Unfortunately, there is no evidence that distributing funds using an on-chain DAO of some kind is going to lead to more positive outcomes. 

What problem does the faucet solve again? Maybe I am misunderstanding something but the faucet decides nothing and adds no transparency. Letting the EF do the coordination could be made just as transparent. Even this misses the point as transparency is not an ends, it is a means to an ends. Making impacts should be the goal. 

If passing through the funds from the EF to some other sub-orgs maximized impact then that too could be implemented by the EF. No untested on-chain governance mechanism required.

What the faucet really changes is:

1) it creates an on-chain governance mechanism that is more likely to be captured by demagogues and all stripes of power hungry self promoters. The role of influence peddlers will increase and funding will go disproportionately not to the most important projects but to the projects with the best marketing.

2) Decentralizing the development budget at this stage is more likely to result in fragmented and incoherent funding policies. For example, funding three projects to 33% completion is likely to generate less impact for the community than getting one project to 100%. 

3) The resources to do due diligence on projects asking for funding will also be fragmented, leading to more shallow due diligence, or worse yet, fake due diligence by influence peddlers. There is always the problem of watching the watchers but them more watchers the community needs to watch the harder this gets.

Doing due diligence on teams and amassing the expertise to predict what investments into public infrastructure are going to yield the highest impact is a hard problem. It requires the coordination of specialists across many fields. Identifying who is a genuine expert and who is a faker is also a hard problem. 

Bottom line: achieving a coherent funding thesis for public infrastructure, doing due diligence on teams, recruiting true expertise into the process and getting projects to critical mass on key projects is going to be harder if multiple pseudo-foundations with no experience need to coordinate their efforts than just one.

So rather than resorting to a discontinuous shift into a completely new & untested on-chain "DAO" structure that has never worked before and that we have zero experience with, in the mid-short term the community will get a lot more bang for our buck by incrementally improving the existing off-chain institution that has already been doing this for a few years until we have some on-chain alternative that has proven itself - at least on a small scale. We are far from exhausting off-chain improvements. The devil is in the details but better to start with the devil we know than a whole swarm of new devils we don't.

**Comment by GriffGreen** [comment]
I tend to agree here... I'm down to DAO-ify all the things... but lets not try to run before we can crawl... and there will be a lot of squabbling about details if we try too much. 

Would rather model after a working solution then trying to build an experiment when there is so much at risk. We get into a bunch of politics if its anything not totally controlled by the EF

Let's use this to incentivize the EF to revamp their Grant transparency, ask them to agree to a sunset for a transition, throw the block reward funds to the EF, and watch the experiments by groups like Aragon, Status, Giveth, MakerDAO, Moloch etc, see what works, then switch it to an onchain governance later.... Ideally using the staking nodes that eth2.0 uses that can follow a masternode type system modeled after DASH and their forks... 

Creating an onchain governance solution with out playing with it in a low risk environment for a while seems risky and borderline arrogant.... not to mention we will never come to a consensus on the details.

**Comment by kronosapiens** [comment]
Articulate as always, but it seems that there is a difference between directing funding to the EF via a minimal on-chain DAO and hard-coding an EF address into a hard fork. It seems to me that the latter move would represent a type of "piercing of the veil" by explicitly encoding a single institution into the protocol. Instituting even a single mediating layer (even something as simple as a multi-sig with membership rules) which allows for some on-chain oversight (and yes, the expression of a modicum of political will) seems as though it would permit the same goal, but more closely within the bounds of existing norms and values of decentralized decision-making. One could argue that this "leveraging of the EF" is less than "minimal viable governance": something more akin to the original mechanical turk, with a human operator hidden under the table turning a crank.

Further, while I would entertain your argument that there is no current clear alternative to the EF for determining allocations, I am less convinced that any fragmenting of funding would lead to sub-par outcomes. If anything, some competition (even if colored with the political) for funding would encourage the organizations to be as judicious as possible lest they lose support and resources. Further, is it necessarily true that a project funded to "completion" will produce more value than a few projects funded in part? Is budgeting an art or a science, inasmuch as the future cannot be predicted? It may be that some scrappiness is desirable; evidence from the startup world does not contradict.

If the choice is really between entrusting resources to untested and faulty mechanisms and what seems a significant departure from Ethereum norms, then perhaps the best course of action _is_ to wait until these mechanisms are more mature.

That said, I can think of two counter-arguments to myself. The first is that there is a precedent for this in the ZCash Founder's Reward, so perhaps the idea that an explicit protocol-level link (even temporarily) to a single organization may have unintended consequences is perhaps overstated. The second is that perhaps the need to coordinate a fork to even _set_ that explicit link represents a sufficient amount of decentralized decision-making, and that another mediating layer is unnecessary. After all, if Ethereum can hard fork to recover stolen funds, it can just as easily hard fork to mint new ones :)

**Comment by lirazsiri** [comment]
> One could argue that this "leveraging of the EF" is less than "minimal viable governance": something more akin to the original mechanical turk, with a human operator hidden under the table turning a crank.

Ethereum has a tradition of pragmatism. It has so far always preferred one bird in the hand than two birds in the bush. The EVM design is naive. Extremely error prone contracts intentionally look like Javascript. Smart contracts max out at 26K. Almost all dapps rely on Infura as a trusted third party. Yada, yada. The question Ethereum answers is not what is the right solution later but what is the best solution right now. Don't worry guys, we'll fix this later. If you look at Ethereum purely from the prism of technological platform it still kind of sucks. What the technology is really doing is serving as the temporary scaffold for a community of developers drawn to the vision. 

Hopefully in the background, assuming carefully husbandry of the community's resources the researchers will be working on getting this all to work without duct tape and at scale so that all those hackers Ethereum recruited finally start revolutionizing everything. 

If it works out we'll get a global opt-in private jurisdiction that unifies the planet through computational enforcement and minimizes the role of geographically exclusive monopolies on the use of force. World peace!

In the meantime, pay no attention to the man behind the curtain.

> If anything, some competition (even if colored with the political) for funding would encourage the organizations to be as judicious as possible lest they lose support and resources.
Ethereum

Possibly, but in the absence of objectively quantifiable value generation they could also be more judicious in manufacturing the subjective perception of value generated. Faking achievement is always easier than the real thing. If you have two "core devs" and one invests 90% into self promotion and 10% into project management and the other invests 90% into project management and 10% into self promotion who will be more likely to get resources if a cool kids DAO is controlling resource allocation?

> Further, is it necessarily true that a project funded to "completion" will produce more value than a few projects funded in part? Is budgeting an art or a science, inasmuch as the future cannot be predicted? It may be that some scrappiness is desirable; evidence from the startup world does not contradict

Scrappiness and the fragmentation of resources are two separate issues. Regarding fragmentation, let's say for argument's sake that all projects are at 100% scrappy efficiency. Given the same resource constraints funding one ultra-scrappy eWASM implementation to completion necessarily generates more value than funding two ultra-scrappy eWASM implementations to 50%. In the former we have a working eWASM implementation, in the latter, two incomplete projects.

Encouraging scrappiness / resource efficiency can be done without fragmentation. With experience and specialist knowledge you could get projects just enough funding to prove they can deliver to milestone 1, and then if they deliver just enough funding to get to milestone 2, etc. Merely splitting up the coordination by doing it through multiple grant giving organs does not by itself encourage efficient resource allocation. For example if we split up the EF into 2 parts would we expect it to be more or less productive? What about splitting into 5 parts? Or 10? At some point these tiny EF shards would be almost completely useless.

**Comment by EzraWeller** [comment]
I agree it makes little sense to abandon stable footing (the EF, other current grant givers), and go all in on an Eth block reward DAO right away -- assuming there are meaningful ways we can act to "improve" the EF grant process. If there are clear, achievable steps we, as non-members of grant giving orgs, can take to significantly improve funding decisions, that's kind of money on the table, and we should probably just do those things. What are they, if they exist? The only suggestion I see here is to "revamp grant transparency." Perhaps I missed a few.

My gut says that improving the current grant giving processes is also going to be difficult and have no guarantee of success, which makes me favor at least starting some kind of DAO experiment simultaneously. I'm admittedly biased towards that view for professional reasons, of course, though.
